src/App.js

```diff
@@ -1,16 +1,20 @@
+import { render } from '@testing-library/react';
 import './App.css';
 
+const multiplier = 0;
+
+
 function App() {
-  
-  chickenClick = () => {
-    console.log('this is:', this);
-  };
+
+  function chickClick() {
+    multiplier = multiplier + 1;
+  }
 
```

Comment:
This function is currently mutating state that is local to the function `chickClick` perhaps you might want to create some shared state. See https://reactjs.org/docs/hooks-state.html for more details

Reply:
I have renamed them to chickRate and setChickRate and now increases by one when the button is clicked